README.md

```diff
@@ -1 +1,3 @@
-# Meshtastic-arduino
\ No newline at end of file
+Driver for interfacing with Meshtastic nodes over WiFi and Serial connections
+
+Author: Mike Schiraldi
```

Comment:
I generally leave attribution to GitHub and credit for commits, as you usually don't want to maintain all contributor names in a readme.

Reply:
The Arduino custom seems to be for authors to sign their work; see for instance all the Adafruit libraries (e.g., https://github.com/adafruit/Adafruit_CircuitPlayground/blob/master/Adafruit_Circuit_Playground.h#L12 or https://github.com/adafruit/RadioHead/blob/master/RadioHead.h#L2 ) and even the Arduino Core (e.g., https://github.com/arduino/ArduinoCore-avr/blob/master/libraries/SPI/src/SPI.h#L2-L5 )

If the list of contributors ever gets to be too big to manage, I'll count my blessings and revisit this decision.

That said, it does look like the library.properties files tend to list the organization as the author, so I'll do that over there.